net/http/roundtrip.go

```diff
@@ -8,49 +8,50 @@ import (
 	"strconv"
 	"strings"
 
-	golem_go_bindings "github.com/golemcloud/golem-go/golem_go_bindings"
```

Comment:
Please update the go test component in the `golem` repo which uses this after we merge this

Reply:
https://github.com/golemcloud/golem-go/pull/8/commits/61513510b90b0f2fa6d9cc42044509322b0a5501
https://github.com/golemcloud/golem-go/pull/8/commits/5901d4fcd50513a1cfd147696cded8ffe62e3cf1